### PR TITLE
[MM-12396] Option to add user to a channel from the profile pop-over (permissions fix)

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary
This fixes a permission issue for the Add User to Channel modal button in the profile popover. The button was not showing up for Team Admins or Channel Admins when toggled in the System Console.

#### Ticket Link
[Github Ticket](https://github.com/mattermost/mattermost-server/issues/9501)
[Jira Ticket](https://mattermost.atlassian.net/browse/MM-12396)

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit tests passed
- [x] Ran `make flow` to ensure type checking passed
- [x] Added or updated unit tests (required for all new features)

[Webapp PR](https://github.com/mattermost/mattermost-webapp/pull/2435)

#### Test Information
This PR was tested on: [Google Chrome 73.0.3683.86, MacOS 10.14.4]

### CODE REVIEW

**Comment by mickmister** [comment]
There's an issue that was caught in the [webapp PR](https://github.com/mattermost/mattermost-webapp/pull/2435) that needs to be fixed here. The memberships that are being looped through need to be filtered by the current team.

**Comment by mickmister** [comment]
Fix has been applied

File: src/selectors/entities/channels.js [hunk 1]
```diff
@@ -566,6 +566,29 @@ export const canManageChannelMembers: (GlobalState) => boolean = createSelector(
     }
 );
 
+// Determine if the user has permissions to manage members in at least one channel of the current team
+export const canManageAnyChannelMembers: (GlobalState) => boolean = createSelector(
```
Comment (jespino) [inline src/selectors/entities/channels.js]: Maybe change the name to `canManageAnyChannelMembersInCurrentTeam`?

Comment (mickmister) [reply src/selectors/entities/channels.js]: 👍 done

**Comment by mickmister** [comment]
Hi @jespino, do you mind giving this another pass when you have the chance?

**Pull request merged**